src/main/webapp/app/layouts/navbar/navbar.component.ts

```diff
@@ -69,6 +73,14 @@ export class NavbarComponent implements OnInit, OnDestroy {
         }
     }
 
+    checkRouterOnNavigationEnd(): void {
```

Comment:
Documentation missing, also the naming could be better as it is not implying that it has something to do with the guided tours. Also I have a bad feeling about a subscription that is initialized in the constructor, should be in ngOnInit instead.

Reply:
@thilo-behnke Thanks for pointing out - I changed that.